Workbooks/Virtual Machines/At-scale Metrics/Key Metrics.workbook

```diff
@@ -171,218 +167,857 @@
       },
       "name": "text - 5",
       "styleSettings": {
-        "margin": "10px 0 0 0"
+        "margin": "20px 0 20px 0"
       }
     },
     {
-      "type": 10,
+      "type": 11,
       "content": {
-        "chartId": "workbookdb19a8d8-91af-44ea-951d-5ffa133b2ebe",
-        "version": "MetricsItem/2.0",
-        "size": 3,
-        "chartType": 0,
-        "resourceIds": [
-          "{VirtualMachines}"
-        ],
-        "timeContext": {
-          "durationMs": 0
-        },
-        "timeContextFromParameter": "TimeRange",
-        "resourceType": "microsoft.compute/virtualmachines",
-        "resourceParameter": "VirtualMachines",
-        "metrics": [
+        "version": "LinkItem/1.0",
+        "style": "tabs",
+        "links": [
           {
-            "namespace": "microsoft.compute/virtualmachines",
-            "metric": "microsoft.compute/virtualmachines--Percentage CPU",
-            "aggregation": 4,
-            "splitBy": null
+            "cellValue": "selectedTab",
+            "linkTarget": "parameter",
+            "linkLabel": "Overview",
+            "subTarget": "Overview",
+            "style": "link",
+            "workbookContext": {},
+            "templateRunContext": {},
+            "alertRuleContext": {}
           },
           {
-            "namespace": "microsoft.compute/virtualmachines",
-            "metric": "microsoft.compute/virtualmachines--Disk Read Bytes",
-            "aggregation": 1,
-            "splitBy": null
+            "cellValue": "selectedTab",
+            "linkTarget": "parameter",
+            "linkLabel": "Key metrics",
+            "subTarget": "Details",
+            "style": "link"
           },
           {
-            "namespace": "microsoft.compute/virtualmachines",
-            "metric": "microsoft.compute/virtualmachines--Disk Write Bytes",
-            "aggregation": 1,
-            "splitBy": null
+            "cellValue": "selectedTab",
+            "linkTarget": "parameter",
+            "linkLabel": "Regions",
+            "subTarget": "Regions",
+            "style": "link",
+            "workbookContext": {},
+            "templateRunContext": {},
+            "alertRuleContext": {}
           },
           {
-            "namespace": "microsoft.compute/virtualmachines",
-            "metric": "microsoft.compute/virtualmachines--Network In Total",
-            "aggregation": 1,
-            "splitBy": null
+            "cellValue": "selectedTab",
+            "linkTarget": "parameter",
+            "linkLabel": "Resource health",
+            "subTarget": "Health",
+            "style": "link"
+          }
+        ]
+      },
+      "name": "links - 6"
```

Comment:
nit: could change group name to "Link tabs"?

Reply:
Fixed